assets/helpers/tracking/guTracking.js

```diff
@@ -16,6 +17,10 @@ const campaigns : {
     `${baselineTestPrefix}_side_menu_become_supporter`,
     `${baselineTestPrefix}_side_menu_subscribe`,
   ],
+  annual_test: [
+    `${annualTestPrefix}_show_annual`,
+    `${annualTestPrefix}_hide_annual`,
```

Comment:
If we're doing this test in support-frontend, will we have multiple campaign code from dotcom? Won't there just be the one, and then we apply the variants once the users arrive here?

Reply:
Oops this was just something I was playing around with, committed by accident!